unified_planning/model/problem_kind.py

```diff
@@ -370,6 +371,12 @@ def intersection(self, oth: "ProblemKind") -> "ProblemKind":
 basic_temporal_kind.set_typing("FLAT_TYPING")
 basic_temporal_kind.set_time("CONTINUOUS_TIME")
 
+process_temporal_kind = ProblemKind(version=LATEST_PROBLEM_KIND_VERSION)
+process_temporal_kind.set_problem_class("ACTION_BASED")
+process_temporal_kind.set_time("PROCESSES")
+process_temporal_kind.set_time("CONTINUOUS_TIME")
```

Comment:
no FLAT_TYPING here?

Reply:
Isn't FLAT_TYPING a restriction on the kind of types we can define (like no hierarchy)? If so, there is no restriction here, or I didn't understand this :)